src/components/ConversationSettings/LobbySettings.vue

```diff
@@ -140,6 +148,33 @@ export default {
 				valueType: 'timestamp',
 			}
 		},
+
+		showRelativeTime() {
+			return this.lobbyTimer
+				&& this.lobbyTimer > Date.now()
+				&& (this.lobbyTimer - Date.now()) < ONE_DAY_IN_MS // less than 24 hours
+		},
+
+		getTimeZone() {
+			if (!this.lobbyTimer) {
+				return ''
+			}
+			const date = new Date(this.lobbyTimer)
```

Comment:
Aligning with the usage of NcDateTimePicker value as in example
![image](https://github.com/nextcloud/spreed/assets/84044328/7e03d102-ff8c-4678-957f-ed7960ebc98f)